qiskit/circuit/library/standard_gates/s.py

```diff
@@ -82,6 +85,9 @@ class SdgGate(Gate):
 
     This is a Clifford gate and a square-root of Pauli-Z.
```

Comment:
I think this statement is actually correct as is because the Paulis are their own inverses so $(S^\dagger)^2=(S^2)^\dagger = Z^\dagger = Z$. I'll correct that wrong equivalence. Thank you for noticing!